scripts/run_server_asylo

```diff
@@ -4,7 +4,7 @@ readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
 # shellcheck source=scripts/common
 source "$SCRIPTS_DIR/common"
 
-readonly GRPC_PORT="${GRPC_PORT:-8888}"
-
 "$SCRIPTS_DIR/build_server_asylo"
-./bazel-bin/oak/server/asylo/oak --grpc_port="${GRPC_PORT}"
+
+# Launching the server requires a `--application=<dir>/config.bin` flag.
+./bazel-bin/oak/server/asylo/asylo_oak_runner "$@"
```

Comment:
@ipetr0v I think the terminology is still a bit confusing, in particular the `run_server_asylo` script runs an `asylo_oak_runner` binary (different order of qualifiers, and runner vs server). Could we resolve this at some point? I would go with `runner` or `loader` everywhere, since conceptually it's running an application, which may or may not have a server component (usually it does).